src/runtime/virtcontainers/clh.go

```diff
@@ -178,8 +178,8 @@ var clhKernelParams = []Param{
 }
 
 var clhDebugKernelParams = []Param{
-	{"console", "ttyS0,115200n8"},     // enable serial console
-	{"systemd.log_target", "console"}, // send loggng to the console
+	{"console", "hvc0"},               // enable console output
```

Comment:
You can use this approach but that means you won't get the kernel logs from before the time `virtio-console` driver is probed.

Reply:
@sboeuf Can you please take a second look and confirm that we won't lose early kernel logs in this way?

Reply:
Also, I wonder do we know why using `serial` as guest console now doesn't capture the guest userland output (given it was working fine at least back to kata 1.x)?

Reply:
@likebreath - I tested Kata version 1.12.1 and version 1.13.0-alpha0 (both using the bundled cloud-hypervisor v0.11.0) and they boot using `console=ttyS0,115200n8 systemd.log_target=console ` with the following CLH config:

```json
"console": {"mode":"Off"},
"serial": {"mode":"Tty"}
```

... but there is no kernel or userland output in the journal when debug is enabled. I wonder if this is related to CLH not using the proxy in 1.x?